receiver/k8sclusterreceiver/internal/pod/pods.go

```diff
@@ -126,6 +128,15 @@ func GetMetadata(pod *corev1.Pod, mc *metadata.Store, logger *zap.Logger) map[ex
 	meta := maps.MergeStringMaps(map[string]string{}, pod.Labels)
 
 	meta[podCreationTime] = pod.CreationTimestamp.Format(time.RFC3339)
+	phase := pod.Status.Phase
+	if phase == "" {
+		phase = corev1.PodUnknown
+	}
+	meta[podPhase] = string(phase)
+	reason := pod.Status.Reason
```

Comment:
This field is going to be always empty. We need to update `Transform` function above to preserve the value.

Reply:
Thanks. Took for granted the receiver's existing usage of this field had this in.